Assets/Scripts/Behaviors/Explosing.cs

```diff
@@ -7,10 +7,8 @@ public class Explosing : ABehavior {
 	public ParticleSystem		explosion;
 	private ParticleSystem		_currentPS;
 	
-	public void Awake() {
-		this._shortName = "Explosing";
-		this._description = "Explose a dispositif";
-	}
+	public string shortName = "Explosing";
```

Comment:
je l'aurais mis en private, en tout cas je pense qu'il faut genre uniformiser les Behaviors genre dans booting t'as fait ca : [SerializeField] private string _shortName = "Booting";
	[SerializeField] private string _description = "This is the starting process for a machine";
Du coup je pense que faut mettre la aussi en private ou sinon mettre en public dans booting mais on est pas censé changer le shortname et la description d'un behavior